CMakeLists.txt

```diff
@@ -65,7 +65,7 @@ if(WITH_JPEG)
     # Check whether the version of libjpeg we found was libjpeg-turbo and print a
     # warning if not.
     set(CMAKE_REQUIRED_LIBRARIES ${JPEG_LIBRARIES})
-    set(CMAKE_REQUIRED_FLAGS -I${JPEG_INCLUDE_DIR})
+    set(CMAKE_REQUIRED_INCLUDES ${JPEG_INCLUDE_DIRS})
```

Comment:
ACK about CMAKE_REQUIRED_INCLUDES, but I fear FindJPEG does not set  JPEG_INCLUDE_DIRS:
https://cmake.org/cmake/help/v3.4/module/FindJPEG.html

Reply:
It was indeed introduced in a later version. Either we can bump the minimum required version or at least switch from CMAKE_REQUIRED_FLAGS to CMAKE_REQUIRED_INCLUDES here.

Reply:
I'd prefer the latter.

Reply:
:cherries: -picked

Reply:
Is in updated diff.

Reply:
picked